phys2bids/interfaces/txt.py

```diff
@@ -14,6 +14,42 @@
 LGR = logging.getLogger(__name__)
 
 
+def multifreq(timeseries, freq):
```

Comment:
Rather than "multifreq", can we call this something like "check_multifreq" or similar?

Reply:
sure